packages/wonder-blocks-button/src/themes/default.ts

```diff
@@ -25,8 +30,6 @@ const theme = {
             primary: {
                 default: tokens.color.white,
                 disabled: tokens.color.offBlack32,
-                // used in boxShadow
-                inverse: tokens.color.darkBlue,
```

Comment:
**question:** When we remove things from a theme, is it considered a breaking change? It seems like it isn't since we don't export the theme from the package explicitly, though thought I'd double check! 

I was also wondering if we needed to remove this from other themes, but I see now that the khanmigo theme doesn't have these set since we merge the default theme with the khanmigo theme!

Reply:
Good question, and you are right. Right now, as theming works, this is only available internally, so it wouldn't be a breaking change.

The other themes are built based on the shape of the `default` object, and that means that if we remove a key that is used in a merged theme (e.g. `khanmigo`), then TS would complain about it :). I created that merge theme architecture having that in mind, so it would help us to have a more robust theming approach.

Reply:
Nice! Good to know this about our theming!